packages/thirdweb/src/extensions/erc721/drops/write/claimTo.test.ts

```diff
@@ -0,0 +1,3 @@
+import { describe } from "vitest";
+
+describe("ERC721 claimTo", () => {});
```

Comment:
remove?

Reply:
okay I'm messing up something with the stack again. unrelated changes from other PRs.

Fixing